stdlib/nodejs/pathname.rb

```diff
@@ -0,0 +1,30 @@
+class Pathname
+
+  @__fs__ = node_require :fs
+  `var __fs__ = #{@__fs__}`
+
+  # Instance Methods
+
+  def initialize(path)
+    #raise ArgumentError if path == "\0"
+    if Pathname === path
+      @path = path.path
+    else
+      @path = path
+    end
+  end
```

Comment:
@Mogztter exactly, probably also fixing the implementation to accept `Pathname` instances too, which I don't think currently works:

https://github.com/opal/opal/blob/0cc85df99943f720124f01a030e23db8f8449ac9/stdlib/pathname.rb#L2-L5